scripts/multi_region_tests.py

```diff
@@ -25,7 +25,8 @@
             "dxCompiler_Sydney",
             "dxCompiler_Azure",
             "dxCompiler_Amsterdam",
-            "dxCompiler_Berlin"]
+            "dxCompiler_Berlin",
+            "dxCompiler_OFH_TRE"]
```

Comment:
Hmm.. It looks like dxCompiler_London and dxCompiler_Europe_London are missing from here.

Reply:
yes, I mentioned that in the APPS-1428. Should I add them?

Reply:
fixed in [dd4730a](https://github.com/dnanexus/dxCompiler/pull/407/commits/dd4730a369b0c731832d98415aa7a5be7e02e0b2)